extras/nest_vars.sh.in

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh
 
-# NEST is installed here. When you relocate NEST, change this variable.
-export NEST_INSTALL_DIR="@CMAKE_INSTALL_PREFIX@"
+# NEST is installed here. When you relocate NEST, change these variables.
+NEST_LIB_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/nest/"
+NEST_BIN_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR/"
 
-# NEST finds standard *.sli files $NEST_DATA_DIR/sli
-export NEST_DATA_DIR="${NEST_INSTALL_DIR}/@CMAKE_INSTALL_DATADIR@"
-
-# NEST finds help files $NEST_DOC_DIR/help
-export NEST_DOC_DIR="${NEST_INSTALL_DIR}/@CMAKE_INSTALL_DOCDIR@"
```

Comment:
Similarly to `NEST_DATA_DIR`, this one already defaults to the value it is set to here and I don't see a problem with removing the export here.